src/index.html

```diff
@@ -7,38 +7,50 @@
     <title>Product cards</title>
     <!-- <link rel="stylesheet" href="./styles/normalize.css"> -->
     <link rel="stylesheet" href="./style.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
     <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
   </head>
   <body class="body">
+
     <div class="card" data-qa="card">
+
       <div class="card__block-img">
         <img
           src="./images/imac.jpeg"
           alt="Monoblock view"
           class="card__monoblock-image"
         >
```

Comment:
i think you dont need div here, only img is enough

Reply:
agree